docusaurus.config.js

```diff
@@ -47,10 +47,17 @@ module.exports = {
         sidebarPath: require.resolve("./optimize_sidebars.js"),
         editUrl: "https://github.com/camunda/camunda-docs/edit/main/",
         versions: {
+          "3.11.0": {
+            label: "8.3 / 3.11.0",
+          },
+          "3.10.0": {
+            banner: "none",
+          },
```

Comment:
This prevents docusaurus from showing an "version is unsupported" banner, because we support previous versions for 18 months.